client/sources/ok_test/models.py

```diff
@@ -100,9 +100,11 @@ def unlock(self, interact):
                                 for case in suite.cases])
         for suite_num, suite in enumerate(self.suites):
             for case_num, case in enumerate(suite.cases):
+                case_id = '{} > Suite {} > Case {}'.format(
```

Comment:
If you are worried about students adding their own tests. Why not just give them a blank test file to put their own stuff in and tell them not to touch the course files? You could even give them extra points or grade them on what is in that file, if learning how to write good tests is part of your course goals.

As for getting a unique identification, would a combination of assignment ID, prompt, and the question ID Albert proposed earlier of "Test name -> Suite # -> Case # -> Prompt #" work? Wasn't that the original plan?